install_config/configuring_openstack.adoc

```diff
@@ -50,160 +88,8 @@ subnet-id = <UUID of the load balancer subnet>
-----
-
-[IMPORTANT]
-====
-When triggering a containerized installation, only the directories of
-*_/etc/origin_* and *_/var/lib/origin_* are mounted to the master and node
-container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
-*_/etc/_*.
-====
-
-[[openstack-configuring-nodes-manually]]
-=== Manually Configuring {product-title} Nodes for OpenStack
-
-Edit the appropriate xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration map] 
-and update the contents of the `*kubeletArguments*` and `*nodeName*` sections:
-
-[source,yaml]
-----
-nodeName:
-  <instance_name> <1>
-
-kubeletArguments:
-  cloud-provider:
-    - "openstack"
-  cloud-config:
-    - "/etc/cloud.conf"
-----
-<1> The RFC1123-compliant OpenStack instance name of the node host.
-
-[NOTE]
-====
-If the `nodeName` does not match the OpenStack instance name, the cloud provider
-integration will not work.
-====
-
-[IMPORTANT]
-====
-When triggering a containerized installation, only the directories of
-*_/etc/origin_* and *_/var/lib/origin_* are mounted to the master and node
-container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
-*_/etc/_*.
-====
-
-[[installation-openstack-playbook]]
-=== Installing {product-title} by Using an Ansible Playbook
-
-[IMPORTANT]
-====
-The OpenStack installation playbook is a Technology Preview feature.
-ifdef::openshift-enterprise[]
-Technology Preview features are not supported with Red Hat production service
-level agreements (SLAs), might not be functionally complete, and Red Hat does
-not recommend to use them for production. These features provide early access to
-upcoming product features, enabling customers to test functionality and provide
-feedback during the development process.
-
-For more information on Red Hat Technology Preview features support scope, see
-https://access.redhat.com/support/offerings/techpreview/.
-endif::[]
-====
-
-To install {product-title} on an existing OpenStack installation, use the
-OpenStack playbook. For more information about the playbook, including detailed
-prerequisites, see
-link:https://github.com/openshift/openshift-ansible/tree/master/playbooks/openstack#installation[the OpenStack Provisioning readme file].
-
-To run the playbook, run the following command:
-
-----
-$ ansible-playbook --user openshift \
-  -i openshift-ansible/playbooks/openstack/inventory.py \
-  -i inventory \
-  openshift-ansible/playbooks/openstack/openshift-cluster/provision_install.yml
-----
-
-[[openstack-applying-configuration-changes]]
-== Applying Configuration Changes
-include::install_config/topics/applying_configuration_changes.adoc[]
-
 [[openstack-configuring-zone-labels]]
-=== Configuring Zone Labels for Dynamically Created OpenStack PVs 
+=== Configuring Zone Labels for Dynamically Created OpenStack PVs
 
 Administrators can configure zone labels for dynamically
 created OpenStack PVs. This option is useful if the OpenStack Cinder zone name
```

Comment:
finish to verify this section. looks good to me

Reply:
i have a oc get pv --show-labels is there something else you looking for here?